Src/Clients/WebUI/Controllers/Storage/ActorsWebUiController.cs

```diff
@@ -16,30 +18,55 @@ public class ActorsWebUiController : BaseController
     {
         private readonly IApiTools _apiTools;
         private readonly IAuthorizeApiTools _authorizeApiTools;
+        private readonly ILogger<ActorsWebUiController> _logger;
 
-        public ActorsWebUiController(IApiTools apiTools, IAuthorizeApiTools authorizeApiTools)
+        public ActorsWebUiController(IApiTools apiTools, IAuthorizeApiTools authorizeApiTools,
+            ILogger<ActorsWebUiController> logger)
         {
             _apiTools = apiTools;
             _authorizeApiTools = authorizeApiTools;
+            _logger = logger;
         }
 
         [HttpGet]
-        public IActionResult Index()
+        public async Task<IActionResult> Index()
         {
+            _logger.LogInformation("Request for access to the actor controller Index action method.");
+
+            try
+            {
+                // Check the viability of the JWT token.
+                var tmp = (await _authorizeApiTools.FetchAsync<List<ActorBindingModel>>(
+                    CommonClientsDefaults.WebApiActorsControllerGetAll, JwtToken)).FirstOrDefault();
+            }
+            catch (AuthenticationException e)
+            {
+                _logger.LogWarning(e, e.Message);
+                return RedirectToAction("Login", "Account");
+            }
+
             return View();
         }
 
-        //[HttpGet]
+        // TODO: Specify attribute with HTTP method.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)